docs/OncoKB-Data-Access.md

```diff
@@ -1,13 +1,23 @@
 # Introduction
 
-A data access token is required for versions `v3.2.4` and greater to use OncoKB. Please review 
-OncoKB [terms](https://www.oncokb.org/terms) and request for [data access](https://www.oncokb.org/dataAccess) 
-if you would like to use OncoKB data with your private cBioPortal instance.
+Using OncoKB does not require a data access token by default. But the instance you are connecting to only includes biological information. 
+If you want to include tumor type summary, therapeutic levels and more, please consider obtaining a license from OncoKB.
 
-# Configuration
+# How to obtain an OncoKB license
+1. Please review OncoKB [terms](https://www.oncokb.org/terms) 
+2. Please request for [data access](https://www.oncokb.org/dataAccess)
+3. You can find your token in your [Account Settings](https://www.oncokb.org/account/settings) after login.
+
+# Set up cBioPortal to include therapeutic information
+
+Following properties can be edited in the `portal.properties` file or set in system variables if you are using docker. 
+- `show.oncokb` should be set to `true`
+- `oncokb.token` should be set to a valid OncoKB access token value
+- `oncokb.public_api.url` should be set to `https://www.oncokb.org`
```

Comment:
Is there a different parameter for authenticated API?

Reply:
@jjgao no, there are only three parameters for oncokb and all listed here. As long as they supply the token, we will attach that to the authentication header.